manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java

```diff
@@ -162,6 +166,12 @@ public Bridge updateBridge(String bridgeId, String customerId, BridgeRequest bri
         return existingBridge;
     }
 
+    private Action resolveErrorHandler(Action errorHandler) {
```

Comment:
@kostola As a suggestion, if the user has requested the `endpoint` error handler, resolve the endpoint for them here and put it into the originally requested `Action` `parameters`.  That means we do it once and the user can retrieve the value from the `parameters` on their `Action`.

WDYT?

Reply:
done :+1: